### DISCUSSION TITLE
How to call default stringifier from custom one?

### DISCUSSION THREAD

**Discussion by cgbahk** [inducer/pudb#520] [Q&A] [answered]
I found custom stringifier is strong feature that makes debugging a lot intuitive for some cases.

I followed guide on custom stringifier in:
- https://documen.tician.de/pudb/usage.html#controlling-how-values-get-shown
- https://github.com/inducer/pudb/blob/dab821fe50/examples/stringifier.py

## Issue
One pain point: I hope to construct custom stringifier that call default stringifier (below) as fallback, https://github.com/inducer/pudb/blob/dab821fe50bb1c86ddfba703e0cfab5f7572369a/pudb/var_view.py#L399

but it is not clear for me how to do that.

Is it possible? Or it would be nice to have such feature.

**Comment by inducer** [comment, marked as answer]
You could simply import the default stringifier, like so:
```py
from pudb.var_view import default_stringifier
```
Or am I missing something?

**Reply by cgbahk** [reply]
I cannot believe why I didn't try this... :sweat_smile:

This just works. Thanks
